### PR TITLE
Pass '--path' option to browserify.

### PR DESCRIPTION
**Pull request by jhytonen**
This is similar to pull requests https://github.com/mantoni/mochify.js/pull/124 and https://github.com/mantoni/mochify.js/pull/83, supporting only the browserify paths option.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "mochify",
-  "version": "2.16.0",
+  "version": "2.16.1",
```
Comment (mantoni) [inline package.json]: Please revert this change. I'll bump the version in the release commit.

**Comment by mantoni** [comment]
Thanks for the pull request. Nicely done. One question: Why is the option called `path` in Mochify and `paths` in Browserify? Shouldn't the names be aligned? The command line option could still be `--path`.

**Comment by jhytonen** [comment]
@mantoni It is mainly because I followed convention. There is already a Browserify option called "extensions", which is "extension" in Mochify. It is right above the line that I added in mochify.js file.

**Comment by mantoni** [comment]
Ok, cool. Please revert the package.json change and squash the commits. :+1:

**Comment by jhytonen** [comment]
@mantoni Done.

**Comment by mantoni** [comment]
Released in `2.17.0`. Thanks!

**Pull request merged**